packages/repository-tests/src/crud/relations/fixtures/models/shipment.model.ts

```diff
@@ -22,10 +22,16 @@ export class Shipment extends Entity {
   })
   id: MixedIdType;
 
+  @property({
+    type: 'number',
+  })
+  shipmentId: number;
+
   @property({type: 'string'})
   name: string;
 
-  @hasMany(() => Order, {keyTo: 'shipment_id'})
+  // keyFrom is not the id property of Shipment
+  @hasMany(() => Order, {keyFrom: 'shipmentId', keyTo: 'shipmentInfo'})
```

Comment:
The point of `shipment_id` was to test it with snake case. Is there somewhere else that tests it?

Reply:
reversed